src/main/scala/uk/gov/nationalarchives/notifications/decoders/ExportStatusDecoder.scala

```diff
@@ -0,0 +1,23 @@
+package uk.gov.nationalarchives.notifications.decoders
+
+import java.util.UUID
+
+import com.amazonaws.services.lambda.runtime.events.SNSEvent
+import io.circe.CursorOp.DownField
+import io.circe.{Decoder, HCursor}
+import io.circe._
+import io.circe.parser._
+import io.circe.generic.auto._
+
+object ExportStatusDecoder {
+  case class ExportStatusEvent(consignmentId: UUID, success: Boolean) extends IncomingEvent
+  case class SNS(Message: String)
+  case class Record(Sns: SNS)
+
+  val decodeExportStatusEvent: Decoder[IncomingEvent] = (c: HCursor) => for {
+    message <- c.downField("Records").as[List[Record]]
+    json <- parse(message.head.Sns.Message)
```

Comment:
Just to check - do we know that it's always safe to just take the first item in the list?

Reply:
Yep, there's only ever one https://aws.amazon.com/sns/faqs/#Reliability